src/node_binding.cc

```diff
@@ -42,6 +42,7 @@
   V(config)                                                                    \
   V(contextify)                                                                \
   V(credentials)                                                               \
+  V(encoding)                                                                  \
```

Comment:
My only concern is there is a `node::encoding` as an enum already. If we are going with this approach, IMHO I think it's the right way, we should rename the `node::encoding` enum to something else to avoid confusion.

Reply:
Oh yeah I guess then it can be encoding_binding just like the namespace. Renaming the enum is a breaking change for the API.